spec/fabricators/review_fabricator.rb

```diff
@@ -0,0 +1,4 @@
+Fabricator(:review) do 
+  rating { (1..5).to_a.sample }
+  content { Faker::Lorem.paragraph(3) }
```

Comment:
To be clear, you're saying join the returned array of paragraphs, correct? So line 2 should be: 
content { Faker::Lorem.paragraph(3).join(" ") }